playbooks/dataiku.yml

```diff
@@ -43,45 +41,77 @@
 
-    - name: Debug output
-      debug:
-        msg: "{{ DATA_DIR }}"
+    - name: Create private directories
+      become: true
+      file:
+        path: "{{dss_data_dirs_location}}"
+        state: directory
+        owner: "{{dss_service_user}}"
+        mode: "u=rwx,g=,o="
 
+    - name: Download DSS
+      become: true
+      become_user: "{{dss_service_user}}"
+      get_url:
+        url: "https://cdn.downloads.dataiku.com/public/studio/{{dss_version}}/dataiku-dss-{{dss_version}}.tar.gz"
+        dest: "{{dss_install_dir_location}}/dataiku-dss-{{dss_version}}.tar.gz"
+
+    - name: Unarchive DSS
+      become: true
+      become_user: "{{dss_service_user}}"
+      unarchive:
+        src: "{{dss_install_dir_location}}/dataiku-dss-{{dss_version}}.tar.gz"
+        dest: "{{dss_install_dir_location}}"
+        creates: "{{dss_install_dir_location}}/dataiku-dss-{{dss_version}}"
+        remote_src: yes
+
+    - name: Install DSS dependencies
+      become: true
+      shell:
+        cmd: |
+          {{dss_install_dir_location}}/dataiku-dss-{{dss_version}}/scripts/install/install-deps.sh -yes 2>&1 > /tmp/dss-install-deps.log
+          touch {{dss_install_dir_location}}/dataiku-dss-{{dss_version}}/scripts/install/DEPS-INSTALLED
+        creates: "{{dss_install_dir_location}}/dataiku-dss-{{dss_version}}/scripts/install/DEPS-INSTALLED"
+    
     - name: Check if dataiku is installed
+      become: true
       stat:
-        path: /data/dataiku/lib
+        path: /home/dataiku/dataiku
       register: dataiku_installed
 
-    - name: I am confused a
-      command: 'whoami'
-      register: myidentity
-
-    - name: my secret identity
-      debug:
-        msg: '{{ myidentity.stdout }}'
-
-    - name: Install Dataiku DSS in
-      shell: "/opt/dataiku/dataiku-dss-10.0.2/installer.sh -d {{ DATA_DIR }} -p 11000 -n -y -P /opt/Python/3.6-base/bin/python3.6"
+    - name: Run the Dataiku DSS installer
+      become: true
+      become_user: "{{dss_service_user}}"
+      command: "{{dss_install_dir_location}}/dataiku-dss-{{dss_version}}/installer.sh -d /home/{{dss_service_user}}/dataiku -p {{port}}"
+      args:
+        creates: "/home/{{dss_service_user}}/{{datadir}}/dss-version.json"
       when: not dataiku_installed.stat.exists
 
-    - name: start dataiku
-      become: false
-      shell: /data/dataiku/bin/dss start
+    - name: Install boot service
+      become: true
+      command: "{{dss_install_dir_location}}/dataiku-dss-{{dss_version}}/scripts/install/install-boot.sh -n {{datadir}} /home/dataiku/{{dss_service_user}} dataiku"
+      args:
+        creates: "/etc/init.d/dataiku.{{datadir}}"
+
+    - name: Start the studio
+      become: true
+      service:
+        name: "dataiku.{{datadir}}"
+        state: started
+        enabled: true
+
+    # - name: start dataiku
+    #   become: true
+    #   become_user: "{{dss_service_user}}"
+    #   shell: /home/dataiku/dataiku/bin/dss start 
 
```

Comment:
Putting it here because I do not get an option to add a comment on line 122: have you considered creating a handlers/main.yml dir/file for the handlers? Though I understand if you rather keep it in the playbook because it's just 1-2.